README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+# hw01
+
+Hello, my name is Adrian Przezdziecki, a rising 4th-year in your *Computing for the Social Sciences* class.
+
+## General Info
+
+* My major is **Public Policy Studies** in the College.
+  * I initially started out as a Physics major before getting into the wonderful world of Social   Science.
+* I actually have experience with R in the geographical science world, particularly working with the *sf* package
+  * [Here](https://github.com/JJiang11/flightpath) is a package a friend and I created for our GIS class that uses an API and various packages to analyze real-time flight paths.
+  * Granted, I already learned something new about R with the ggplot package reading, as I would just use general plot() functions in R before.
```

Comment:
Glad to here that. I think `ggplot` is really useful :) Still, I think my favorite things about R beyond base R are `tidyverse` and piping.

Reply:
Just to make it cooler, you can make your "plot()" here become code text: `plot()`